### ISSUE TITLE
Code (CSS) does not wrap and hides under mini map

### ISSUE THREAD

**Issue by mikael1000** [microsoft/vscode#112154]
![2020-12-09_22-54-57](https://user-images.githubusercontent.com/12203516/101693386-d25da500-3a71-11eb-8ff5-d460814caf0f.png)

- VSCode Version: 1.5.1.1
- OS Version: Windows_NT x64 6.3.9600 (win8.1)

Steps to Reproduce:

**First** I did reset the settings to default for the mini map. Then:

**1.**
Open a CSS-file and write code. Code does not wrap for mini map and hides under. It wraps when it hit the side of the window.

**2.**
If I just write characters in the file it wraps when it hits the minimap. It also wraps HTML-code in .html and same for JS. So it seems to be a CSS specific thing.

Does this issue occur when all extensions are disabled?: Yes/No
Yes!

**Comment by aeschli** [comment]
Can you add a screenshot?

**Comment by mikael1000** [comment]
> Can you add a screenshot?

Screenshot added in OP.

**Comment by alexdima** [comment]
@mikael1000 Are you perhaps missing some steps or settings in the issue? Why would you expect for the lines to wrap? Did you configure wrapping? If yes, with what exact settings ?

**Comment by mikael1000** [comment]
@alexdima
Yes I have changed the setting:
**Editor: Word Wrap** and I set it to **on** (default is **off**).

Two reasons I expect it to wrap:

**1.**
Bad to have code hidden under the minimap. Makes it hard to follow the hidden code and a lot of errors are made.

**2.**
The code wraps for HTML and JavaScript in relation to the minimap. I haven't tried any other languages so I don't know.

**Comment by alexdima** [comment]
Thank you for the extra information. This is caused by the css color boxes, which are not taken into account when computing wrapping points. Let's track in #32856
